Description: Disable test that needs a tty
 See terminal_size create-terminal-for-testing.patch for a workaround.
Last-Update: 2023-09-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/tty/unix.rs
+++ b/src/tty/unix.rs
@@ -57,7 +57,7 @@
 }
 
 #[cfg(not(target_os = "redox"))]
-#[test]
+//#[test]
 /// Compare with the output of `stty size`
 fn compare_with_stty() {
     use std::process::Command;
